meta-mel/recipes-graphics/mesa-demos/mesa-demos_%.bbappend

```diff
@@ -0,0 +1,2 @@
+bindir_mel = "/opt/${PN}"
+EXTRA_OECONF_append_mel = " --bindir=${bindir}"
```

Comment:
This line too is pointless, as it's already passed by autotools.bbclass. Just set bindir and move on.